framework/src/multiapps/MultiApp.C

```diff
@@ -144,6 +145,15 @@ validParams<MultiApp>()
       "Additional command line arguments to pass to the sub apps. If one set is provided the "
       "arguments are applied to all, otherwise there must be a set for each sub app.");
 
+  params.addRangeCheckedParam<Real>("relaxation_factor",
+                                    1.0,
+                                    "relaxation_factor>0 & relaxation_factor<2",
+                                    "Fraction of newly computed value to keep."
+                                    "Set between 0 and 2.");
+  params.addParam<std::vector<std::string>>("relaxed_variables",
+                                            std::vector<std::string>(),
+                                            "List of variables to relax during Picard Iteration");
+
```

Comment:
Why is this here?

Reply:
It is complicated. Let us say we have a master + subapp + subapp. When we do Picard relaxation, we can
1. do relaxation on master;
2. do relaxation on subapp as a master of another subapp;
3. do relaxation on subapp as subapp of its master.
This is used to control Case 3.